### DISCUSSION TITLE
[WinUI] Why is Dispatcher not marked as Deprecated?

### DISCUSSION THREAD

**Discussion by Diyari-Kurdi** [microsoft/microsoft-ui-xaml#8400] [Q&A] [answered]
Hello,
Why is **Dispatcher** not deprecated in **WinUI 3**, even though it is always **NULL**?

**Comment by ghost1372** [comment, marked as answer]
https://github.com/microsoft/microsoft-ui-xaml/discussions/8381#discussioncomment-5579291
